modules.d/90dmsquash-live/dmsquash-live-root.sh

```diff
@@ -343,6 +375,15 @@ if [ -n "$FSIMG" ]; then
             unpack_archive $FSIMG /run/initramfs/fsimg/
         fi
         FSIMG=/run/initramfs/fsimg/rootfs.img
+
+        if [ -n ${join} ]; then
```

Comment:
Did you intend to support the 'writeable_fsimg' option?  If so, please verify that this new code path works.